sys/net/crosslayer/pkt/pkt.c

```diff
@@ -66,4 +95,85 @@ void pkt_hlist_remove(pkt_hlist_t **list, pkt_hlist_t *header)
 
 }
 
+int pkt_add_header(pkt_t *pkt, void *header_data, pktsize_t header_len,
+                   pkt_proto_t header_proto)
+{
+    pkt_hlist_t header, *ptr;
+
+#ifdef DEVELHELP
+
+    if (pkt == NULL) {
```

Comment:
This is conflicting with what @OlegHahm said, back when I introduced this in #1129. Also: Did you mean `s/puts/DEBUG/`?